### DISCUSSION TITLE
What happens if there is a network issue while uploading VOD files to an S3 bucket?

### DISCUSSION THREAD

**Discussion by cloud-tentosoft** [ant-media/Ant-Media-Server#6021] [FAQs]
1) Will the locally stored VOD be uploaded to S3 once the network is stable?
2) RTMP endpoint also disconnects. I can see the RTMP endpoint in the settings, but it is not restreamed after the network becomes stable.

**Comment by USAMAWIZARD** [comment]
Hi @MuhammadYaqoobG , 
let me try this on my end and get back to you.

**Reply by USAMAWIZARD** [reply]
@MuhammadYaqoobG I have tried the s3 senario it can upload the video after the network is stable again.

**Reply by cloud-tentosoft** [reply]
I stored video clips using a REST API, capturing them every 5 minutes. When I called the 'record stop' API to conclude the recording session, it attempted to upload the clips to the S3 bucket. However, if the network was down at the time of the API call, the upload was unsuccessful. Subsequently, when the network connection was restored, calling the 'record stop' API would initiate the upload process for at that time. Unfortunately, the video file that was stopped when the network was down remained in the local folder and wasn't uploaded.

**Reply by USAMAWIZARD** [reply]
which version of Ams are you using right now

This is how I tried to reproduce the issue on my end.

1. Start a live stream 
2. Video is getting recorded
3. turn off internet 
4. send request to   stop live stream
5. video recording finish
6. try to upload without internet
7. trun on internet 
8. video file gets uploaded

**Reply by cloud-tentosoft** [reply]
2.7.0
